### ISSUE TITLE
Associate a domain name with the Github domain

### ISSUE THREAD

**Issue by CliffordAnderson** [HeardLibrary/twine-for-wikicite#68]
Maybe "wikicite4librarians"?

**Comment by baskaufs** [comment]
Whenever we decide on this, I will change the name of the Twine story to it. That causes it to appear at the top of the navigation page. For the new test splash page version, I did this, so you see "wikicite4librarians" at the top of the nav page (instead of "Learn Wikidata", which is on the current index.html). 

Also, on the splash page, we need some kind of reference to WikiCite. If rando people discover it, they might be scratching their heads since most of the content is generic enough that people wouldn't necessarily pick up that this if focused on librarians working in the arena of WikiCite.

See also #70 where I raised the issue of where we should link to our grant proposal and to our GitHub site. At the bottom of the splash? On the credits page? At the bottom of the nav page?
